include/up-cpp/communication/NotificationSink.h

```diff
@@ -31,9 +31,11 @@ namespace uprotocol::communication {
 /// receiving half of the notification model.
 struct NotificationSink {
 	using ListenCallback = transport::UTransport::ListenCallback;
-
+	using UTransportSharedPtr =
+	    std::shared_ptr<uprotocol::transport::UTransport>;
 	using SinkOrStatus =
 	    utils::Expected<std::unique_ptr<NotificationSink>, v1::UStatus>;
+	using ListenHandle = uprotocol::transport::UTransport::ListenHandle;
```

Comment:
Same comment as in #179:
```suggestion
	using SinkOrStatus =
	    utils::Expected<std::unique_ptr<NotificationSink>, v1::UStatus>;
	using ListenHandle = transport::UTransport::ListenHandle;
```